scripts/install_and_authenticate_to_gcloud.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+# Install and Gcloud and authenticate on travis
+
+export BOTO_CONFIG=/dev/null;
```

Comment:
At some point somebody should probably leave a comment as to why we would do something like BOTO_CONFIG, as this is going to obfuscate the branch where somebody made that change in the git logs...